envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyExtension.kt

```diff
@@ -9,9 +9,10 @@ import pl.allegro.tech.servicemesh.envoycontrol.config.RandomConfigFile
 import pl.allegro.tech.servicemesh.envoycontrol.config.containers.EchoServiceExtension
 import pl.allegro.tech.servicemesh.envoycontrol.logger
 
-class EnvoyExtension(envoyControl: EnvoyControlExtension,
-                     val localService: EchoServiceExtension? = null)
-    : BeforeAllCallback, AfterAllCallback {
+class EnvoyExtension(
+    envoyControl: EnvoyControlExtension,
+    val localService: EchoServiceExtension? = null
```

Comment:
Why public val?

Reply:
True, once I introduced the Ingress/Egress operations it no longer has to be public.